### PR TITLE
[doc] updated "prerequisites" with more detail

### PR DESCRIPTION
**Pull request by alexzhc**
Responding to https://github.com/hwameistor/hwameistor/issues/155, we have updated the prerequisites for Quick Start with more information about os, software, and hardware.

### CODE REVIEW

File: docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md [hunk 1]
```diff
@@ -5,8 +5,44 @@ sidebar_label: "Prerequisites"
...
-- Have at least one unused drive
+1. Kubernetes version: `1.18+`
+1. CoreDNS is deployed
+
+## Host
+
+### Linux Distribution
+
+1. CentOS `7.5+`
+1. Kylin `V10`
```
Comment (yankay) [inline docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md:18]: Do we support Ubuntu, Rockey Linux ,RHEL .

File: docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md [hunk 2]
```diff
@@ -5,8 +5,44 @@ sidebar_label: "Prerequisites"
...
+
+### Processor Architecture
+
+1. x86_64
+1. ARM64
+
+### Package Dependencies
+
+1. The latest `LVM2`
```
Comment (yankay) [inline docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md:27]: Should we change to the latest `LVM2` installed ?

File: docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md [hunk 3]
```diff
@@ -5,8 +5,44 @@ sidebar_label: "Prerequisites"
...
+
+For test, each host must have at least one unused drive with a minimal size of 10GiB.
+
+For production, it is recommended to have a least one unused drive, protected by RAID1 or RAID5/6, with a minimal size of 200GiB.
+
+### Network
+
+For test, a simple 1Giga TCP/IP network is adequate.
+
+For production, and especially with HA features, it is recommended to have a redundant 10Giga TCP/IP network.
```
Comment (yankay) [inline docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md:48]: Recommended to have a redundant 10Giga TCP/IP network, if enable the HA features .

**Comment by alexzhc** [comment]
closed for complicated conflicts.

**Pull request closed without merging**